src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/FilterSet.cs

```diff
@@ -80,6 +81,7 @@ static FilterSet()
             PropertyFilter = CreateCompletionFilterAndAddToBuilder(FeaturesResources.Properties, 'p', WellKnownTags.Property);
             MethodFilter = CreateCompletionFilterAndAddToBuilder(FeaturesResources.Methods, 'm', WellKnownTags.Method);
             ExtensionMethodFilter = CreateCompletionFilterAndAddToBuilder(FeaturesResources.Extension_methods, 'x', WellKnownTags.ExtensionMethod);
+            OperatorFilter = CreateCompletionFilterAndAddToBuilder(FeaturesResources.operator_, 'r', WellKnownTags.Operator);
```

Comment:
`FeaturesResources.operator_` is probably wrong here. It should be pluralized:  `FeaturesResources.Operators`, but this resource does not exist. Should I add?

Reply:
Definitely :)

Reply:
Done (9b4cbe2)